### PR TITLE
huly 0.6.333 (new cask)

### PR DESCRIPTION
**Pull request by chari-md**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/h/huly.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  url "https://dist.huly.io/Huly-macos-#{version}-#{arch}.zip"
+  name "Huly"
+  desc "All-in-One Project Management Platform"
+  homepage "https://huly.io/"
+
+  livecheck do
+    url "https://github.com/hcengineering/platform/releases"
+    strategy :github_latest
+  end
```
Comment (bevanjkay) [inline Casks/h/huly.rb:16]:
```suggestion
  livecheck do
    url "https://dist.huly.io/latest-mac.yml"
    strategy :electron_builder
  end
```

File: Casks/h/huly.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  depends_on macos: ">= :catalina"
```
Comment (bevanjkay) [inline Casks/h/huly.rb:18]:
```suggestion
  auto_updates true
  depends_on macos: ">= :catalina"

File: Casks/h/huly.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 arm:   "067fc376b35db1240f1ca79829386ecc74bccd14fb699e44745764edc21ee467",
+         intel: "85256fe6af1a8b777de72ef5c3d9cff65d86cf1731d7f2a9ec58eafbe733f4aa"
+
+  url "https://dist.huly.io/Huly-macos-#{version}-#{arch}.zip"
+  name "Huly"
+  desc "All-in-One Project Management Platform"
+  homepage "https://huly.io/"
+
+  livecheck do
+    url "https://dist.huly.io/latest-mac.yml"
```
Comment (bevanjkay) [inline Casks/h/huly.rb:14]: Looks like the app actually uses a slightly different pattern...

```
    url "https://dist.huly.io/huly-mac.yml"

Comment (marcellodomenis) [reply Casks/h/huly.rb:14]: If you check in their [tags](https://github.com/hcengineering/platform/tags) there is a 0.6.334 version. I don't know if you prefer to:
-  keep `latest-mac.yml` and later `bump-cask-pr` that should resolve the errors
- or use `huly-mac.yml` which seems to follow their [releases](https://github.com/hcengineering/platform/releases)

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @marcellodomenis

**Pull request merged**